specs/core/0_beacon-chain.md

```diff
@@ -237,8 +236,8 @@ These configurations are updated for releases, but may be out of sync during `de
 | `BASE_REWARD_QUOTIENT` | `2**5` (= 32) |
 | `WHISTLEBLOWING_REWARD_QUOTIENT` | `2**9` (= 512) |
 | `PROPOSER_REWARD_QUOTIENT` | `2**3` (= 8) |
-| `INACTIVITY_PENALTY_QUOTIENT` | `2**24` (= 16,777,216) |
-| `MIN_PENALTY_QUOTIENT` | `2**5` (= 32) |
+| `INACTIVITY_PENALTY_QUOTIENT` | `2**25` (= 33,554,432) |
+| `MIN_SLASHING_PENALTY_QUOTIENT` | `2**5` (= 32) |
 
 * **The `BASE_REWARD_QUOTIENT` is NOT final. Once all other protocol details are finalized it will be adjusted, to target a theoretical maximum total issuance of `2**21` ETH per year if `2**27` ETH is validating (and therefore `2**20` per year if `2**25` ETH is validating, etc etc)**
 * The `INACTIVITY_PENALTY_QUOTIENT` equals `INVERSE_SQRT_E_DROP_TIME**2` where `INVERSE_SQRT_E_DROP_TIME := 2**12 epochs` (~18 days) is the time it takes the inactivity penalty to reduce the balance of non-participating [validators](#dfn-validator) to about `1/sqrt(e) ~= 60.6%`. Indeed, the balance retained by offline [validators](#dfn-validator) after `n` epochs is about `(1 - 1/INACTIVITY_PENALTY_QUOTIENT)**(n**2/2)` so after `INVERSE_SQRT_E_DROP_TIME` epochs it is roughly `(1 - 1/INACTIVITY_PENALTY_QUOTIENT)**(INACTIVITY_PENALTY_QUOTIENT/2) ~= 1/sqrt(e)`.
```

Comment:
If `INACTIVITY_PENALTY_QUOTIENT` changes from `2**25` to `2**24`, then `INACTIVITY_PENALTY_QUOTIENT != INVERSE_SQRT_E_DROP_TIME**2` now. This sentence needs to be updated or we change `INACTIVITY_PENALTY_QUOTIENT` back to `2**24` with `INACTIVITY_PENALTY_QUOTIENT // 2` in formula.

Reply:
I think this sentence should go away. We should have a separate document explaining design decisions/rationales. (This will be included in the Transparent Paper, whenever I get round to writing it.)

Reply:
Yeah, it's also mentioned in @vbuterin's https://notes.ethereum.org/s/rkhCgQteN#Slashing-and-anti-correlation-penalties. We could (i) update rationales doc (ii) remove this sentence from the spec and link to rationale doc for now.